### DISCUSSION TITLE
porting UI to wxWidgets

### DISCUSSION THREAD

**Discussion by MarleenDev** [WinMerge/winmerge#2379] [Ideas]
Hello,
Has anyone ever tried porting the UI to wxWidgets? This way it could become portable to other platforms like linux or mac (there may be other dependencies that need changing, but one has to start somewhere)
I'm not sure I have time to do this, but it seems an interesting thing to do.

**Comment by sdottaka** [comment]
From what I can see, no one does.

There was a project to port WinMerge to QT, but development has stopped.

https://github.com/OzzieIsaacs/winmerge-qt

It would probably take several years of development to port WinMerge to wxWidgets.

It might be quicker to implement the features you want in WinMerge for an existing cross-platform diff/merge tool.
